frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/NavigationMenu/InputPopover/InputPopover.test.tsx

```diff
@@ -59,22 +80,10 @@ describe('InputPopover', () => {
     expect(mockSaveNewName).toHaveBeenCalledTimes(0);
   });
 
-  it('saves the new name on Enter key press', async () => {
```

Comment:
Do not we need this test ?

Reply:
Not anymore. As you can see in the ```InputPopover.tsx``` file, the ```handleKeyPress``` function is removed

Reply:
I believe the users still expects it to work when pressing enter, so we should keep this test, as Jamal points out. If it's not working without the `handleKeyPress` function, we should add it back.

Reply:
Then I'll add it back. That was part of the issue when using the popover, but now as we use the confirm it works fine.